csharp/ql/src/semmle/code/csharp/commons/Assertions.qll

```diff
@@ -9,11 +9,13 @@ private import ControlFlow::BasicBlocks
 
 /** An assertion method. */
 abstract class AssertMethod extends Method {
-  /** Gets the index of the parameter being asserted. */
-  abstract int getAssertionIndex();
+  /** Gets the index of a parameter being asserted. */
+  abstract int getAnAssertionIndex();
```

Comment:
We should add deprecated `getAssertionIndex()`, `getAssertedParameter()`, and `getExpr()` predicates for backwards compatibility.